samples/net/updatehub/prj.conf

```diff
@@ -0,0 +1,25 @@
+# Turn on the updatehub
+CONFIG_UPDATEHUB=y
+
+#Example of product_uid, you must set you product uid.
+CONFIG_UPDATEHUB_PRODUCT_UID="e4d37cfe6ec48a2d069cc0bbb8b078677e9a0d8df3a027c4d8ea131130c4265f"
```

Comment:
In order to avoid the user to modify this file if he changes the product uid, this setting could be placed in a separate overlay config file. This way this config file would only contain generic settings which are not related to user environment.

The CONFIG_UPDATEHUB_SERVER="<your-addres-ip>" could be in the same overlay file.